Assets/Scripts/Battlefield/CreatureScripts/CharacterFactory.cs

```diff
@@ -80,10 +79,18 @@ void Awake()
 
                 numUnits += 2;
             }
+        }
 
-
+        private IEnumerable<IUnit> GetAllUnits() 
+        {
+            Debug.LogWarning("No squad ID set, defaulting to all units");
+            DatabaseConnection conn = new DatabaseConnection();
+            DatabaseReader reader = conn.QueryAllFromTable("Units");
+            while (reader.NextRow())
+            {
+                yield return new Unit(reader.GetIntFromCol("ID"));
+            }
+            yield return null;
         }
```

Comment:
Use the Unit.GetAllUnits() instead of rewriting it

Reply:
Ah ok, I was just using what was there before but that is better

Reply:
Resolved in 40c6f71